flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/runtime/stream/table/AggregateITCase.scala

```diff
@@ -259,6 +259,30 @@ class AggregateITCase(mode: StateBackendMode) extends StreamingWithStateTestBase
     assertEquals(expected.sorted, sink.getRetractResults.sorted)
   }
 
+  @Test
+  def testListAggWithExpression():Unit = {
```

Comment:
Could we integrate the test into the other test cases, e.g. testGroupAggregateWithExpression to avoid introducing an extra IT case?